From 49c5cbc510517570abfb8a5881b5b971a15a83cc Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Thu, 24 Sep 2015 11:30:21 +0800
Subject: [PATCH 0579/5242] MLK-11615 ARM: dts: imx6qp-sabresd: Enable
 prefetch for the four mxcfbs

commit  554340250677bda9f5cd66281b913bb9896f62d9 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch enables the prefetch feature for the four mxcfbs.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6qp-sabresd.dts |   16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/arch/arm/boot/dts/imx6qp-sabresd.dts b/arch/arm/boot/dts/imx6qp-sabresd.dts
index 3864021..e77ad3c 100644
--- a/arch/arm/boot/dts/imx6qp-sabresd.dts
+++ b/arch/arm/boot/dts/imx6qp-sabresd.dts
@@ -54,6 +54,22 @@
 	status = "disabled";
 };
 
+&mxcfb1 {
+	prefetch;
+};
+
+&mxcfb2 {
+	prefetch;
+};
+
+&mxcfb3 {
+	prefetch;
+};
+
+&mxcfb4 {
+	prefetch;
+};
+
 &pre1 {
 	status = "okay";
 };
-- 
1.7.9.5

